zjit/src/hir.rs

```diff
@@ -3634,11 +3633,10 @@ mod tests {
         ");
         assert_method_hir_with_opcode("test", YARVINSN_setinstancevariable, expect![[r#"
             fn test:
-            bb0():
-              v1:Fixnum[1] = Const Value(1)
-              v3:BasicObject = PutSelf
-              SetIvar v3, :@foo, v1
-              Return v1
+            bb0(v0:BasicObject, v4):
```

Comment:
this bare `v4` should have a type

Reply:
fixed by https://github.com/ruby/ruby/pull/13529/commits/2e3f4b2863de9f5cefd09e978a8eb9d99fe38543